examples/navier-stokes/navierstokes.c

```diff
@@ -880,6 +904,7 @@ int main(int argc, char **argv) {
   CeedVectorRestoreArray(multlvec, &mult);
 
   // Destroy mult vecs
```

Comment:
Looks good. Something very minor. I would update the comment here with `// Destroy nodes and mult vecs` or something along those lines